src/bp/ui-admin/src/Pages/Confusion/details.jsx

```diff
@@ -132,23 +128,20 @@ const MatrixComponent = props => {
 
   return (
     <div>
-      <div>
-        <table className="matrix">{rows}</table>
-      </div>
+      <div style={{ textAlign: 'left' }}>Context: {context}</div>
+      <table className="matrix">{rows}</table>
     </div>
   )
 }
 
-const Details = data => {
-  return (
-    <div>
-      <div className="App">
-        {_.map(data, (matrix, index) => (
-          <MatrixComponent matrix={matrix} name={index} />
-        ))}
-      </div>
+const Details = data => (
+  <div>
+    <div className="App">
+      {_.chain(data.data)
```

Comment:
is the _.chain really helpfull here?

Reply:
Yep, but it's not so easy to see why :/
If you chain a dictionnary, the map will have the signature (val, key), that's how I get the matrix and the context at the same time. Object.entries could do the same though.

Reply:
:+1:

Reply:
_.map, or Object.entries 
Let's keep chain for chaining ;)